CMSgov/mint-app
# [NOREF] Notifications Center UX Review

# NOREF


## Changes and Description
UX Review: https://docs.google.com/document/d/1kuHkKtP2v4hoK7GgcPAM1vSVA5NUW4WfSRi7LLOodB0/edit

- Changed Mark all as read functionality
- Add success message for notification setting change
- Fixed spacing within notification text and link
- Fixed weird quotation issue found in smaller screens



## How to test this change

1. Create a number of notifications (replying to discussion, tag in dicusssion, share a plan, etc) 
2. Verify problem areas listed in [UX Review Doc](https://docs.google.com/document/d/1kuHkKtP2v4hoK7GgcPAM1vSVA5NUW4WfSRi7LLOodB0/edit) has been addressed



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
